docs/yaml/yaml.go

```diff
@@ -220,10 +220,10 @@ func parseMDContent(mdString string) (description string, examples string) {
 	parsedContent := strings.Split(mdString, "\n## ")
 	for _, s := range parsedContent {
 		if strings.Index(s, "Description") == 0 {
-			description = strings.Trim(s, "Description\n")
+			description = strings.TrimSpace(strings.TrimPrefix(s, "Description"))
 		}
 		if strings.Index(s, "Examples") == 0 {
-			examples = strings.Trim(s, "Examples\n")
+			examples = strings.TrimSpace(strings.TrimPrefix(s, "Examples"))
```

Comment:
actually, looking at this again. I think the `TrimSpace()` wont do what you want, because there are characters after this line.  I think it still needs to have the trailing `\n` in the TrimPrefix

The contents of `s` is a multiline string that includes the paragraphs after the header.

Reply:
It will first remove the `Examples` word; after that, the line is empty (or contains whitespace only); trimming the whitespace then cleans up all whitespace surrounding the section; https://play.golang.org/p/XrM2WQWCjlA